profiles/static/js/croppie.js

```diff
@@ -0,0 +1 @@
lt:e(function(e){var t=e.points,n=document.createElement("div"),i=document.createElement("img"),o=t[2]-t[0],r=t[3]-t[1];return d(n,"croppie-result"),n.appendChild(i),p(i,{left:-1*t[0]+"px",top:-1*t[1]+"px"}),i.src=e.url,p(n,{width:o+"px",height:r+"px"}),n}.call(t,n))}})}function D(e){if(!this.options.useCanvas||!this.options.enableOrientation)throw"Croppie: Cannot rotate without enableOrientation && EXIF.js included";var t,n,i,o,r,l=this,u=l.elements.canvas;l.data.orientation=(t=l.data.orientation,n=e,i=a.indexOf(t)>-1?a:s,o=i.indexOf(t),r=n/90%i.length,i[(i.length+o+r%i.length)%i.length]),x(u,l.elements.img,l.data.orientation),k.call(l),_.call(l),copy=null}if(window.jQuery){var T=window.jQuery;T.fn.croppie=function(e){if("string"===typeof e){var t=Array.prototype.slice.call(arguments,1),n=T(this).data("croppie");return"get"===e?n.get():"result"===e?n.result.apply(n,t):"bind"===e?n.bind.apply(n,t):this.each(function(){var n=T(this).data("croppie");if(n){var i=n[e];if(!T.isFunction(i))throw"Croppie "+e+" method not found";i.apply(n,t),"destroy"===e&&T(this).removeData("croppie")}})}return this.each(function(){var t=new q(this,e);t.$=T,T(this).data("croppie",t)})}}function q(e,t){if(e.className.indexOf("croppie-container")>-1)throw new Error("Croppie: Can't initialize croppie more than once");if(this.element=e,this.options=u(c(q.defaults),t),"img"===this.element.tagName.toLowerCase()){var n=this.element;d(n,"cr-original-image"),m(n,{"aria-hidden":"true",alt:""});var i=document.createElement("div");this.element.parentNode.appendChild(i),i.appendChild(n),this.element=i,this.options.url=this.options.url||n.src}if(C.call(this),this.options.url){var o={url:this.options.url,points:this.options.points};delete this.options.url,delete this.options.points,O.call(this,o)}}q.defaults={viewport:{width:100,height:100,type:"square"},boundary:{},orientationControls:{enabled:!0,leftClass:"",rightClass:""},resizeControls:{width:!0,height:!0},customClass:"",showZoomer:!0,enableZoom:!0,enableResize:!1,mouseWheelZoom:!0,enableExif:!1,enforceBoundary:!0,enableOrientation:!1,enableKeyMovement:!0,update:function(){}},q.globals={translate:"translate3d"},u(q.prototype,{bind:function(e,t){return O.call(this,e,t)},get:function(){var e=H.call(this),t=e.points;return this.options.relative&&(t[0]/=this.elements.img.naturalWidth/100,t[1]/=this.elements.img.naturalHeight/100,t[2]/=this.elements.img.naturalWidth/100,t[3]/=this.elements.img.naturalHeight/100),e},result:function(e){return P.call(this,e)},refresh:function(){return function(){W.call(this)}.call(this)},setZoom:function(e){L.call(this,e),h(this.elements.zoomer)},rotate:function(e){D.call(this,e)},destroy:function(){return function(){var e,t,n=this;n.element.removeChild(n.elements.boundary),e=n.element,t="croppie-container",e.classList?e.classList.remove(t):e.className=e.className.replace(t,""),n.options.enableZoom&&n.element.removeChild(n.elements.zoomerWrap),delete n.elements}.call(this)}}),e.Croppie=window.Croppie=q});
```

Comment:
Why are we storing a vendored library in our repository? We can get it from a cdn: https://cdnjs.com/libraries/croppie
Ideally we would move to using something like yarn or npm to grab out packages, but a cdn is preferable to storing vendored code

Reply:
done it